src/main/scala/com/ebiznext/comet/schema/generator/Yml2DDLJob.scala

```diff
@@ -75,13 +77,16 @@ class Yml2DDLJob(config: Yml2DDLConfig, schemaHandler: SchemaHandler)(implicit
             )
           case None => Map.empty[String, (TableRemarks, Columns, PrimaryKeys)]
         }
-        val oldTables = existingTables.keys.filter(table =>
+        val oldTables = existingTables.keys.filterNot(table =>
           schemas.map(_.name.toLowerCase()).contains(table.toLowerCase())
         )
+
         oldTables.foreach { table =>
-          val schema = schemas
-            .find(_.name.toLowerCase() == table.toLowerCase())
-            .getOrElse(throw new Exception("Should never happen"))
```

Comment:
This can happen in the case where a table exists in the db but not in the schema definition.
Updated behavior: drop tables that exist in the db but are not in the schema definition.

Reply:
LGTM